odml/section.py

```diff
@@ -27,7 +27,10 @@ class BaseSection(base.Sectionable):
 
     :param name: string providing the name of the Section. If the name is not
                  provided, the uuid of the Property is assigned as its name.
+                 Section name is a required attribute.
```

Comment:
"... the object id of the Section..."

Reply:
Copy paste at its best it seems.

Reply:
Does not show up as updated since the line below is highlighted. Also changed it in `odml/base.py`.